provisioning-app/ocp-config/Tailorfile

```diff
@@ -0,0 +1,8 @@
+namespace cd
+selector app=prov-app
+param-file ../../../ods-configuration/ods-core.env
+ignore-path bc:/spec/output/to/name
+ignore-path bc:/spec/output/imageLabels
```

Comment:
As there is no `BuildConfig` resource, we should be able to remove those ignored paths.